components/brave_today/common/features.cc

```diff
@@ -12,8 +12,16 @@ namespace features {
 
 const base::Feature kBraveNewsFeature{"BraveNews",
                                       base::FEATURE_ENABLED_BY_DEFAULT};
-const base::Feature kBraveNewsV2Feature{"BraveNewsV2",
-                                        base::FEATURE_DISABLED_BY_DEFAULT};
+
+const base::Feature kBraveNewsV2Feature {
+  "BraveNewsV2",
+#if BUILDFLAG(IS_ANDROID)
+      base::FEATURE_DISABLED_BY_DEFAULT
```

Comment:
Doesn't this mean a bunch of our unittests are going to fail on Android?

Reply:
I think they should be fine since we set the flag to enabled or disabled on each relevant test.